### ISSUE TITLE
Failed to activate: Cannot mix BigInt and other types, use explicit conversions

### ISSUE THREAD

**Issue by UziTech** [file-icons/atom#843]
[Enter steps to reproduce:]

1. upgrade to Atom 1.56.0-beta0

**Atom**: 1.56.0-beta0 x64
**Electron**: 9.4.4
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [file-icons](https://github.com/file-icons/atom) package 2.1.46

### Stack Trace

Failed to activate the file-icons package

```
At Cannot mix BigInt and other types, use explicit conversions

TypeError: Cannot mix BigInt and other types, use explicit conversions
    at BigIntStats._checkModeProperty (internal/fs/utils.js:364:21)
    at BigIntStats.StatsBase.isDirectory (internal/fs/utils.js:287:15)
    at FileSystem.resolveType (/packages/file-icons/node_modules/atom-fs/lib/filesystem.js:110:28)
    at FileSystem.get (/packages/file-icons/node_modules/atom-fs/lib/filesystem.js:69:13)
    at /packages/file-icons/lib/ui.js:34:29
    at Function.simpleDispatch (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:1182710)
    at Emitter.emit (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:1184151)
    at /packages/file-icons/lib/ui.js:61:19
    at Workspace.observeTextEditors (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:430973)
    at UI.observe (/packages/file-icons/lib/ui.js:56:19)
    at Object.activate (/packages/file-icons/lib/main.js:14:11)
    at Package.activateNow (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:3485232)
    at ~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:3484644
    at Package.measure (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:3482222)
    at ~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:3484504
    at new Promise (<anonymous>)
    at Package.activate (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:3484447)
    at PackageManager.activatePackage (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:385022)
    at ~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:384638
    at Config.transactAsync (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:334166)
    at PackageManager.activatePackages (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:384588)
    at PackageManager.activate (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:11:384138)
    at ~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:1:824962
    at async Promise.all (index 0)
    at AtomEnvironment.startEditorWindow (~/AppData/Local/atom-beta/app-1.56.0-beta0/resources/app/static/<embedded>:1:825447)
```

### Commands

```
```

### Non-Core Packages

```
atom-beautify 1.0.0 
atom-debug-ui 1.0.4 
atom-ide-datatip 0.24.0 
atom-ide-debugger-node 0.7.3 
atom-ide-definitions 0.4.1 
atom-ide-hyperclick 1.0.9 
atom-ide-markdown-service 2.0.0 
atom-ide-outline 2.0.0 
atom-ide-signature-help 0.15.3 
atom-ide-ui 0.13.0 
atom-live-server 2.3.0 
atom-minify 0.8.0 
atom-music 2.0.3 
atom-npm-outdated 0.19.0 
atom-overtype-mode 0.4.0 
atom-path-intellisense 1.2.2 
atom-ternjs 0.20.0 
auto-detect-indentation 1.3.0 
autoclose-html-plus 0.27.2 
autocomplete-emojis 2.5.0 
autocomplete-html-entities 0.2.0 
autocomplete-json 5.5.5 
autocomplete-paths 2.13.0 
bracket-colorizer 1.4.0 
busy-signal 2.0.1 
change-case 0.6.5 
color-picker 2.3.0 
context-menu-remove 0.2.1 
cson 2.0.1 
cursor-history 0.14.0 
dbclick-tree-view 1.6.3 
dbl-click-select-inside-brackets 1.0.0 
dec-hex-oct-bin 0.1.4 
docblockr 0.13.7 
double-tag 1.7.0 
editor-stats 0.17.0 
emmet 2.4.3 
eslint-format 1.0.23 
external-open 0.6.3 
file-icons 2.1.46 
file-watcher 2.0.0 
flex-tool-bar 2.2.5 
git-menu 3.2.8 
git-time-machine 2.1.0 
highlight-selected 0.17.0 
hyperlink-hyperclick 3.0.2 
ide-python 1.7.3 
ide-typescript 0.9.1 
indent-guide-improved 1.4.13 
intentions 1.1.5 
language-babel 2.85.0 
language-batch 0.4.0 
language-diff 0.7.0 
language-htaccess 0.2.0 
language-ini 1.23.0 
language-json5 1.1.1 
language-latex 1.2.0 
language-peg-pegjs 0.1.0 
language-powershell 5.0.0 
language-regexp 1.1.3 
line-diff-details-plus 1.9.1 
line-ending-converter 1.3.2 
linter 3.3.0 
linter-coffeelint 2.0.71 
linter-coverage 0.3.0 
linter-eslint 8.5.5 
linter-htmlhint 1.6.3 
linter-js-standard 7.0.0 
linter-js-yaml 1.4.6 
linter-jsonlint 1.4.0 
linter-markdown 5.2.11 
linter-phpcs 1.7.6 
linter-sass-lint 1.9.0 
linter-stylelint 4.5.1 
linter-ui-default 3.2.5 
markdown-preview-plus 4.9.0 
markdown-to-pdf 0.1.1 
minimap 4.39.8 
minimap-bookmarks 0.4.6 
minimap-cursorline 1.0.2 
minimap-find-and-replace 5.0.8 
minimap-git-diff 4.3.6 
minimap-highlight-selected 4.6.5 
minimap-lens 0.8.2 
minimap-linter 2.2.2 
minimap-pigments 0.2.2 
minimap-selection 4.5.4 
minimap-split-diff 0.3.7 
notifications-plus 2.0.18 
notifications-plus-confetti 0.2.0 
open 2.0.5 
open-terminal-here 2.3.2 
php-ide-serenata 5.4.0 
pigments 0.40.6 
project-config 1.0.2 
project-manager 3.3.8 
remote-ftp 2.2.4 
sass-autocompile 0.13.5 
scroll-editor-on-middle-click 0.7.0 
split-diff 1.6.1 
Sublime-Style-Column-Selection 1.7.5 
sync-settings 5.2.7 
sync-settings-git-location 1.1.11 
tabs-to-spaces 1.0.5 
teletype 0.13.4 
temp-find-and-replace 0.219.3 
terminal-commands 1.4.0 
termination 0.7.6 
terminus 4.1.2 
text-manipulation 0.6.0 
themed-context-menu 0.2.2 
todo-show 2.3.2 
tool-bar 1.4.2 
trailing-semicolon 1.1.0 
trailing-spaces 0.4.0 
uzitech-toggle-quotes 0.0.0 
vic 0.0.0 
wakatime 9.0.2 
web-accessibility-checker 0.1.4 
x-terminal 13.0.1 
```

**Comment by nesc58** [comment]
Same here on windows 10

```
Atom: 1.57.0-nightly0 x64
Electron: 9.4.4
Chrome: 83.0.4103.122
Node: v12.14.1
```

**Comment by ahmedgadit** [comment]
Same here on windows 10...

**Comment by Alhadis** [comment]
Fixed in [`atom-fs@v0.2.1`](https://github.com/file-icons/atom-fs/releases/tag/v0.2.1), which will be part of a patch release I'm currently working on. Hold tight.
